### PR TITLE
Normalization

### PR DESCRIPTION
**Pull request by mihirsamdarshi**
I finished adding the gray threshold slider, and have formatted to follow current GRNsight standards

### CODE REVIEW

File: web-client/public/js/upload.js [hunk 1]
```diff
@@ -102,18 +102,24 @@ $(function () {
...
+    grayThreshold = true;
+    displayNetwork(currentNetwork, name, normalization, grayThreshold);
+  });
 
-  var displayNetwork = function (network, name, normalization) {
+  var displayNetwork = function (network, name, normalization, grayThreshold) {
 
     currentNetwork = network;
     console.log(network); // Display the network in the console
     console.log(normalization);
```
Comment (eileenchoe) [inline web-client/public/js/upload.js]: We might want to be leaving more informative console.logs, if we are going to be including them at all. Even just a console.log("normalization: " + normalization); would be sufficient

File: web-client/public/js/upload.js [hunk 2]
```diff
@@ -102,18 +102,24 @@ $(function () {
...
+    console.log(grayThreshold);
```
Comment (eileenchoe) [inline web-client/public/js/upload.js]: Same goes here

**Comment by eileenchoe** [comment]
@kdahlquist do we want the slider values for the grey threshold to be a percentage or a decimal? Currently they are being displayed as a decimal.

**Comment by eileenchoe** [comment]
Pull Request Checklist
- [ ] All Tests Pass
- [ ] All Loaded Demos look as Expected
- [ ] Reload works as expected
- [ ] Loading File with Error brings up error modal, and it looks as expected
- [ ] Loading File with Warning brings up warnings modal
- [ ] Viewport Option Changing works as expected
- [ ] Viewport Size Changing works as expected
- [ ] Hide/Show Weights works as expected
- [ ] Force Sliders change the number above them and have an effect on the graph
- [ ] Locking/Unlocking/Resetting/Undoing the force graph parameters works as expected
- [ ] Weighted graph loaded with the "default to black edges" option checked appears as expected
- [ ] Import works for both SIF and GraphML
- [ ] Graph can be exported to SIF and GraphML

**Comment by kdahlquist** [comment]
Hmm.  Good question.  I can see an argument for either way.  For example, zoom is using percentages, but the gravity slider is using a decimal between 0 and 1.  I just asked @nikavafadari and Nick (no Github handle) and they voted for percent.  Is this easy to change?

**Comment by mihirsamdarshi** [comment]
- [ ] All Tests Pass
- [ ] All Loaded Demos look as Expected
- [ ] Reload works as expected
- [ ] Loading File with Error brings up error modal, and it looks as expected
- [ ] Loading File with Warning brings up warnings modal
- [ ] Viewport Option Changing works as expected
- [ ] Viewport Size Changing works as expected
- [ ] Hide/Show Weights works as expected
- [ ] Force Sliders change the number above them and have an effect on the graph
- [ ] Locking/Unlocking/Resetting/Undoing the force graph parameters works as expected
- [ ] Weighted graph loaded with the "default to black edges" option checked appears as expected
- [ ] Import works for both SIF and GraphML
- [ ] Graph can be exported to SIF and GraphML

**Comment by eileenchoe** [comment]
@mihirsamdarshi is still working on the requested changes, so this pull request will be reviewed again tomorrow

**Comment by kdahlquist** [comment]
So, when this PR is merged, @mihirsamdarshi and @eileenchoe will merge the normalization branch and deal with the merge conflicts?  This sounds like a good task for tomorrow.

**Comment by eileenchoe** [comment]
Yes, that's the plan!

**Comment by dondi** [comment]
This looks good. I was going to mark up a bunch of style issues but realized that this is pre-linting. Presumably, once this is merged, and `normalization` is being merged into `beta`, the linter will catch a bunch of issues and those can be fixed at that time.

**Pull request merged**